src/renderer/contexts/BitcoinContext.tsx

```diff
@@ -23,7 +24,8 @@ export type BitcoinContextValue = {
   pushTx: typeof pushTx
   reloadFees: typeof reloadFees
   resetTx: typeof resetTx
-  ledgerAddress$: typeof ledgerAddress$
+  ledgerMainnetAddress$: typeof ledgerMainnetAddress$
+  ledgerTestnetAddress$: typeof ledgerTestnetAddress$
```

Comment:
You might need just one ledgerAddress$ , which depends on `network$`

Reply:
@veado not exactly. i thought about it but then realized that we need to reconnect ledger advice for EVERY network changing. what do you think ?

Reply:
see https://github.com/thorchain/asgardex-electron/pull/639#discussion_r533324044